src/components/signup/SignUpPage.js

```diff
@@ -40,10 +58,21 @@ class SignUp extends React.Component {
                 <div className="row social-buttons-row">
 
                   <div className="col-lg-4 social-button">
-                    <button type="button" className="btn btn-primary w-100 google-btn">
-                      <i className="fab fa-google mr-2" />
-                      Signup with Google
-                    </button>
+                    <GoogleLogin
+                      clientId={process.env.GOOGLE_CLIENT_ID}
+                      // eslint-disable-next-line react/jsx-no-bind
+                      render={renderProps => (
+                        <button
+                        onClick={renderProps.onClick} type="button" className="btn btn-primary w-100 google-btn">
+                          <i className="fab fa-google mr-2" />
+                          Signup with Google
+                        </button>
+                      )}
+                      buttonText="Signup with Google"
+                      onSuccess={this.responseGoogle}
+                      onFailure={this.responseGoogle}
+                      onRequest={request}
+                    />
```

Comment:
isn't the exact code for the login and signup google button being repeated  on the two pages, the only difference being the text on the button.

I think the button can be extracted into a standalone component and imported here to follow the DRY principle.